src/styles/components/_carousel.scss

```diff
@@ -1,5 +1,39 @@
 $mc-carousel-centered-padding: 100px !default;
 
+.slick-dots {
+  /* stylelint-disable */
+  display: flex !important;
```

Comment:
I hate having to use `!important` here, but slick slider inlines `style="display: block;"` when the dots should be visible.